library/alloc/src/collections/btree/map.rs

```diff
@@ -2738,22 +2740,21 @@ impl<K, V> Clone for Cursor<'_, K, V> {
 #[unstable(feature = "btree_cursors", issue = "107540")]
 impl<K: Debug, V: Debug> Debug for Cursor<'_, K, V> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_tuple("Cursor").field(&self.key_value()).finish()
+        f.write_str("Cursor")
```

Comment:
Not a requirement to merge, but it would be nice if this also detailed where the cursor was relative to the elements in the map.

Reply:
Maybe by including the value from either side of the cursor, if any?

```console
Cursor {
    prev: Some("m"),
    next: Some("n"),
}
```